src/db_utils.py

```
@@ -0,0 +1,175 @@
+ユーティリティ関数を提供します。
+"""
+
+import os
+import threading
+from typing import Optional
+from contextlib import contextmanager
+from sqlalchemy import create_engine, Engine, text
+from sqlalchemy.orm import sessionmaker, Session
+from dotenv import load_dotenv
+
+from .models import Base
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)